test/Benchmarks/src/Vector/Vector.enso

```diff
@@ -32,6 +32,7 @@ main =
     Bench.measure ((random_vec.drop (Last 20)).sum) "Drop Last 20 and Sum" iter_size num_iterations
     Bench.measure (random_vec.filter (x -> x % 3 == 1)) "Filter" iter_size num_iterations
     Bench.measure (random_vec.filter_with_index (i->x -> (i+x) % 3 == 1)) "Filter With Index" iter_size num_iterations
+    Bench.measure (random_vec . map (x -> x +  random_gen.nextLong) . filter (x -> x % 3 == 1)) "Map & Filter" iter_size num_iterations
```

Comment:
Out of curiosity: what is the reason for adding _this_ particular pair? I imagine we can pair many operations like that.